server.js

```diff
@@ -69,6 +135,52 @@ function LocationResult(search, location) {
   this.longitude = location.body.results[0].geometry.location.lng;
 }
 
+// Location helper function to check the database for location information
+function getLocation(request, response) {
+  Location.lookupLocation({
+    tableName: Location.tableName,
+
+    query : request.query.data,
+
+    // If the location exists, send it
+    cacheHit: function(result) {
+      response.send(result);
+    },
+
+    // If the location doesn't exist, request it from the API, save it in database, and send it to the client
+    cacheMiss: function() {
+      const url = `https://maps.googleapis.com/maps/api/geocode/json?address=${this.query}&key=${process.env.GOOGLE_API_KEY}`;
+
+      return superagent.get(url)
+        .then(result => {
+          const location = new Location(this.query, result);
+          // We need a .then() becasue we want to wait for the id to be returned before sending the location object to the client
+          // If we semd the location object back before we receive the id from the database, the other APIs will not know what teh request.query.data.id is and it will not be undefined
+          location.save()
+            .then(location => response.send(location));
+        })
+        .catch(error => processError(error));
+    }
+  })
+}
+
+function singleLookup = (options) => {
```

Comment:
This function is trying to be both a regular 'function' and an arrow function, so the syntax is causing the app to crash. How would you rewrite this line to be either a regular function or an arrow function?